extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/DefaultTokenStateManager.java

```diff
@@ -126,4 +129,30 @@ private static String getRefreshTokenCookieName(OidcTenantConfig oidcConfig) {
         String cookieSuffix = CodeAuthenticationMechanism.getCookieSuffix(oidcConfig);
         return SESSION_RT_COOKIE_NAME + cookieSuffix;
     }
+
+    private String encryptToken(String token, OidcTenantConfig oidcConfig) {
+        if (oidcConfig.tokenStateManager.encryptionRequired.orElse(false)) {
+            String encryptionSecret = oidcConfig.tokenStateManager.encryptionSecret
```

Comment:
Wouldn't be better to calculate the enc-key at build time? Falling back to the secret accordingly?